libraries/plugins/witness/witness.cpp

```diff
@@ -41,21 +41,21 @@ namespace bpo = boost::program_options;
 
 void new_chain_banner( const graphene::chain::database& db )
 {
-   std::cerr << "\n"
+   ilog("\n"
       "********************************\n"
       "*                              *\n"
       "*   ------- NEW CHAIN ------   *\n"
       "*   - Welcome to Graphene! -   *\n"
       "*   ------------------------   *\n"
       "*                              *\n"
       "********************************\n"
-      "\n";
+      "\n");
    if( db.get_slot_at_time( fc::time_point::now() ) > 200 )
    {
-      std::cerr << "Your genesis seems to have an old timestamp\n"
+      wlog("Your genesis seems to have an old timestamp\n"
```

Comment:
Personally I think having `\n` in the middle of a log message is a bit strange. What's your opinion?